swagger/swagger.json

```diff
@@ -11,6 +11,30 @@
   } ],
```

Comment:
Don't commit the swagger for now. The new endpoint shouldn't show up in it when you move it. We'll have a later story to add to the swagger file.